README.md

```diff
@@ -52,25 +52,14 @@ modify (`/usr/local`) to being editable by your user (sudo chown -R $USER /usr/l
 
 - [Grunt](http://gruntjs.com/): Install globally with `npm install -g grunt-cli`
 
-- [Typescript](http://www.typescriptlang.org/): Install globally with  `npm install -g typescript`
-
-    - This is assuming you have `ruby` and `rubygems` installed.
-
-
 ### Setup of uProxy codebase
 
-1. Clone uProxy and its submodules (and its submodules' submodules...):
-`git clone https://github.com/uProxy/uProxy.git`
-or `git clone git@github.com:uProxy/uproxy.git` if you have your ssh access to github set up (useful if you use 2-step auth for github, which you should do).
+ 1. Clone uProxy and its submodules (and its submodules' submodules...): `git clone https://github.com/uProxy/uProxy.git` or `git clone git@github.com:uProxy/uproxy.git` if you have your ssh access to github set up (useful if you use 2-step auth for github, which you should do).
 
-2. In the uProxy repository's root directory, run `bower install` to install any bower dependencies.
-
-3. In the uProxy repository's root directory, run `npm install`. This will install all local dependencies,
-as appropriate to run in Chrome and Firefox. The first time you run this, you'll see lots of npm, bower and grunt messages. Check the last couple of lines in case there is an error.
+ 1. In the uProxy repository's root directory, run `npm install`. This will install all local dependencies, as appropriate to run in Chrome and Firefox. The first time you run this, you'll see lots of npm, bower and grunt messages. Check the last couple of lines in case there is an error.
```

Comment:
Nope, look at the rendered diff -- turns out once you ident these numbered lists the renderer will automatically number them for you.